### DISCUSSION TITLE
help wanted: ollama-webui didn't start successfully

### DISCUSSION THREAD

**Discussion by TheStarAlight** [open-webui/open-webui#674] [General]
**Problem**
Hi! I'm lack in docker experience, and I find it difficult to start ollama-webui successfully.
Currently I'm using Manjaro Linux on my remote workstation and I installed docker and docker compose. I installed the docker image and it seems that the docker started successfully, but when I'm trying to access the port 3000, there's no reply (I used the vscode ssh to forward port 3000 on my remote workstation to my computer, but the 1234 port is forwarded successfully, so the vscode should not be blamed). I'm completely a greenhand so I just don't know what do do.
Can anyone help? Thank you for your attention :D Please contact me if you need more information. 😄

**Additional context**
The printed msg:
```
$ docker images
REPOSITORY                          TAG       IMAGE ID       CREATED        SIZE
ghcr.io/ollama-webui/ollama-webui   main      4800a0dc9697   26 hours ago   2.03GB
hello-world                         latest    d2c94e258dcb   9 months ago   13.3kB
```
```
$ docker run 4800a0dc9697
No WEBUI_SECRET_KEY provided
Generating WEBUI_SECRET_KEY
Loading WEBUI_SECRET_KEY from .webui_secret_key
INFO:     Started server process [1]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)
^C INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [1]
```

**Comment by officialsahyaboutorabi** [comment]
I am also experiencing the same error. I have tried reinstalling it several times. Still didn't work.

**Reply by tjbck** [reply]
Cannot access the webui at all? Please share more details with us!

**Reply by officialsahyaboutorabi** [reply]
No WEBUI_SECRET_KEY provided
Generating WEBUI_SECRET_KEY
Loading WEBUI_SECRET_KEY from .webui_secret_key
INFO:     Started server process [1]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)

**Comment by ghost** [comment]
The command in your description `docker run 4800a0dc9697` doesn't publish any container ports nor share the network with the host OS. Try adding `-p 3000:8080` (this forwards port 3000 into the container to port 8080) or `--network=host` (this option exposes the port as 8080 not 3000) to your command. Please read the [documentation](https://docs.openwebui.com/#quick-start-with-docker-).

**Reply by TheStarAlight** [reply]
@lainedfles Thank you for your reply! Now I understand what was wrong and I succeed in running the server :D Thanks very much!
